registration/include/pcl/registration/impl/sample_consensus_prerejective.hpp

```diff
@@ -217,14 +217,13 @@ SampleConsensusPrerejective<PointSource, PointTarget, FeatureT>::computeTransfor
 
   // Temporaries
   std::vector<int> inliers;
-  float inlier_fraction;
   float error;
-
+  
   // If guess is not the Identity matrix we check it
   if (!guess.isApprox (Eigen::Matrix4f::Identity (), 0.01f))
   {
-    getFitness (inliers, error);
-    inlier_fraction = static_cast<float> (inliers.size ()) / static_cast<float> (input_->size ());
+    getFitness (guess, inliers, error);
+    float inlier_fraction = static_cast<float> (inliers.size ()) / static_cast<float> (input_->size ());
```

Comment:
const

Aside: are we checking for `input_->size()` somewhere?

Reply:
Done

It seems there is input size check for ```setInputTarget``` but not for ```setInputSource``` in ```pcl::Registration```. Indeed, I didn't see errors/warnings when I fed an empty cloud to ```setInputSource```. Should we open another PR to add a size check for ```setInputSource``` in ```pcl::Registration```?

https://github.com/PointCloudLibrary/pcl/blob/3906e52ee7ae4f817aef2f9e4aa6091634f12ed5/registration/include/pcl/registration/impl/registration.hpp#L47-L53
https://github.com/PointCloudLibrary/pcl/blob/3906e52ee7ae4f817aef2f9e4aa6091634f12ed5/registration/include/pcl/registration/registration.h#L190-L194